backend/main.go

```diff
@@ -231,17 +242,33 @@ func main() {
 	router := mux.NewRouter()
 	router.HandleFunc(newrelic.WrapHandleFunc(app, "/ws", hub.Handler))
 	router.HandleFunc(newrelic.WrapHandleFunc(app, "/download/{path:.*}", nomad.downloadFile))
-	router.PathPrefix("/static").Handler(http.FileServer(myAssetFS))
 	router.HandleFunc(newrelic.WrapHandleFunc(app, "/config.js", func(w http.ResponseWriter, r *http.Request) {
 		response := make([]string, 0)
 		response = append(response, fmt.Sprintf("window.NOMAD_READ_ONLY=%s", strconv.FormatBool(cfg.ReadOnly)))
 		response = append(response, fmt.Sprintf("window.NOMAD_ADDR=\"%s\"", cfg.Address))
 		response = append(response, fmt.Sprintf("window.NOMAD_LOG_LEVEL=\"%s\"", cfg.LogLevel))
 
+		var endpointURL *url.URL
+		var err error
+		if cfg.ProxyAddress != "" {
+			endpointURL, err = url.Parse(cfg.ProxyAddress)
+			if err != nil {
+				logger.Infof("Error parsing supplied web.proxy-address of %s: %s", cfg.ProxyAddress, err)
```

Comment:
One thing I'm thinking of doing is just skipping the URL splitting mainly because most things in the code that depend on the url either 1) are using the currently loaded scheme anyway (i.e. in `sagas/event.js` and the React `main.js`) or have been removed anyway in favour of relative links. So I think we can just get away with getting rid of this parsing together (I was probably being a bit overzealous) and just expect the proxy URL to just be something like `example.com/nomad` without the scheme supplied.

I'll rebase this with the updated code tomorrow!